File: daneel/daneel/services/code_analysis/analysis/test_fixtures/simple.c
#include <stdio.h>

// Structure definition since C doesn't have classes
struct Person
{
    char *name;
    int age;
};

// Function that works with the struct
void print_person(struct Person p)
{
    printf("Name: %s, Age: %d\n", p.name, p.age);
}

File: daneel/daneel/services/code_analysis/analysis/test_fixtures/simple.go
package main

import "fmt"

// Structure definition
type Person struct {
	name string
	age  int
}

// Method (using receiver)
func (p Person) Print() {
	fmt.Printf("Name: %s, Age: %d\n", p.name, p.age)
}

// Standalone function
func Add(a, b int) int {
	return a + b
}
